### DISCUSSION TITLE
Creating lowercase file names

### DISCUSSION THREAD

**Discussion by martinpallmann** [nvim-telekasten/telekasten.nvim#310] [Q&A] [answered]
Hi, I'm a noob to neovim, lua and telekasten. Is there a way to configure the filenames in the following way?

The title is: `This is my title` and this should be the generated file name: `this-is-my-title.md`,

I already found the config param `filename_space_subst` which takes care of the spaces. I need now something to lowercase the title. Help is greatly appreciated.

**Comment by lambtho12** [comment, marked as answer]
It is the other way around actually if I am not mistaken. 
Type `this is my title` when prompted for a title and it should be capitalized in your note `# This is my title`, while the filename is small caps `this-is-my-title.md`.

**Reply by martinpallmann** [reply]
Hm. Ok. That solves it somehow. It's just unfortunate that I'm German and we tend to capitalize nouns. So that would mean I'd still need to edit the title. But at least I can use this without hurting too much. Thank you for the answer.

**Reply by lambtho12** [reply]
Arf, indeed then there is no straightforward way to do it.
We should add a `filename_smallcaps` option then and some logic to convert it properly so it works out-of-the box for you.

**Reply by lambtho12** [reply]
I created issue #311 to keep track of it. I am not sure if I will have time to solve this this week. You can have a go at it, it is a rather straightforward change and I will gladly review and merge the PR asap.
